lib/analytical/modules/segment.rb

```diff
@@ -0,0 +1,130 @@
+module Analytical
+	module Modules
```

Comment:
I don't know if the problem is from the GitHub's diff but it seems you are using tabs when you actually should use (2) spaces for indentation across the whole file.